base/shell/explorer/traywnd.cpp

```diff
@@ -264,6 +264,115 @@ class CStartButton
 
 };
 
+static const WCHAR szTrayShowDesktopButton[] = L"TrayShowDesktopButtonWClass";
+
+// The 'Show Desktop' button at edge of taskbar
+class CTrayShowDesktopButton :
+    public CWindowImpl<CTrayShowDesktopButton, CWindow, CControlWinTraits>
+{
+    LONG m_nClickedTime;
+
+public:
+    DECLARE_WND_CLASS_EX(szTrayShowDesktopButton, CS_HREDRAW | CS_VREDRAW, COLOR_3DFACE)
+
+    CTrayShowDesktopButton() : m_nClickedTime(0)
+    {
+    }
+
+    INT WidthOrHeight() const
+    {
+        INT cxy = GetSystemMetrics(SM_CXEDGE);
+        return (cxy ? (2 * cxy) : 6);
```

Comment:
What is "6" ? Also what would happen if SM_CXEDGE returned "1" (i.e. a very small value)?

Reply:
I meant `"6"` is a minimum width or height. Fixed.

Reply:
6:
![6](https://user-images.githubusercontent.com/2107452/191441200-be6addba-8b01-421e-8c98-9018caf089ee.png)

5:
![5](https://user-images.githubusercontent.com/2107452/191441202-c415ac0c-a349-448e-a018-d7374088be8d.png)

4:
![4](https://user-images.githubusercontent.com/2107452/191441209-23725a74-a124-439b-beaf-2a1350594f6e.png)

3:
![3](https://user-images.githubusercontent.com/2107452/191441175-59ed4fb1-8715-4c34-b1e1-b4ed8b4e35bd.png)

Which is the best choice?

Reply:
I'd say 3 or 4 is fine

Reply:
Applied 3.